### DISCUSSION TITLE
[Survey] [Frameworks] Vue, Angular, React

### DISCUSSION THREAD

**Discussion by JimmyBastos** [frontendbr/forum#981] [Dicas]
Galera, estou fazendo uma pequena [**PESQUISA**](https://goo.gl/forms/PHNb9HZE8eiAia2A2) , sobre **Vue**, **Angular** e **React**, para complementar um trabalho/relatório da faculdade. 

Poderiam responder, por favor? São só 5 clicks... kkkk

[UPDATE]

![deepin screenshot_selecionar area_20180310095032](https://user-images.githubusercontent.com/17859531/37242459-a6f70380-2448-11e8-8c27-fc662a4932a4.png)

**Comment by felipemfp** [comment]
![image](https://user-images.githubusercontent.com/8146112/37235250-fd9ce5d2-23db-11e8-971e-aa9787116a1b.png)

Eu acho que essa escala tá invertida (ou pelo menos tenho essa impressão). Se a pergunta é "o quanto é fácil de aprender", 5 deveria ser "muito fácil".

**Comment by JimmyBastos** [comment]
Faz sentido, mas agora não posso alterar. Tem 43 respostas
Talvez possa inverter a pergunta...  

[UPDATE]

Sim, é possível mudar a pergunta sem "quebrar" o formulário. Obrigado pela dica!!

**Comment by danilovaz** [comment]
Faltou Ember.js aí ❤️  😝 !

Só consegui responder 1 pergunta: Outro framework 😢

**Comment by tarciozemel** [comment]
@JimmyBastos Talvez isso ajude com o trabalho: https://research.hackerrank.com/developer-skills/2018/

**Comment by eberhara** [comment]
@JimmyBastos respondido! 
só cuida os termos utilizados porque, além de react ser uma library (não um framework), existem várias discussões sobre o assunto.

**Comment by linconkusunoki** [comment]
@JimmyBastos também recomendaria este trabalho como um norte, já tem a pesquisa no Brasil também

Geral
https://stateofjs.com/2017/front-end/results/

Aqui você encontra no Brasil
https://stateofjs.com/2017/front-end/worldwide/

**Comment by JimmyBastos** [comment]
@linconkusunoki Já conhecia, mas mesmo assim obrigado!! :1st_place_medal:

**Comment by felipefialho** [comment]
Respondido :)

**Comment by mahenrique94** [comment]
Respondido :)

**Comment by matheuseabra** [comment]
De cara aqui que o Vue tá sendo mais utilizado que React, louco! Vou já escrever algum código com esse framework ver qual papo.
